index.js

```diff
@@ -135,14 +135,21 @@ class Ky {
 		throwHttpErrors = true,
 		searchParams,
 		json,
+		retry = {},
 		...otherOptions
 	}) {
 		this._retryCount = 0;
 
 		this._options = {
 			method: 'get',
 			credentials: 'same-origin', // TODO: This can be removed when the spec change is implemented in all browsers. Context: https://www.chromestatus.com/feature/4539473312350208
-			retry: 2,
+			retry: {
+				retries: 2,
+				methods: retryMethods,
+				statusCodes: retryStatusCodes,
+				afterStatusCodes: retryAfterStatusCodes,
```

Comment:
This doesn't seem to be documented. And I find the option names confusing because the difference between `retry.statusCodes` and `retry.afterStatusCodes` is not clear.

That said, I don't have any good suggestions, either.

Reply:
@sindresorhus Should this be exposed?

Reply:
> Should this be exposed?

No

Reply:
@sindresorhus Do you mean you shouldn't be able to overwrite `afterStatusCodes` and just use the defaults?

Reply:
Yes. We have not exposed it in Got either and no one has requested it yet.